crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -315,13 +315,21 @@ module sui::coin {
         deny_list.v2_remove(DENY_LIST_COIN_INDEX, ty, addr, ctx)
     }
 
-    public(package) fun deny_list_v2_most_recent_contains<T>(
+    public(package) fun deny_list_v2_contains_current_epoch<T>(
         deny_list: &DenyList,
         addr: address,
         ctx: &TxContext,
     ): bool {
         let ty = type_name::get_with_original_ids<T>().into_string().into_bytes();
-        deny_list.v2_most_recent_contains(DENY_LIST_COIN_INDEX, ty, addr, ctx)
+        deny_list.v2_contains_current_epoch(DENY_LIST_COIN_INDEX, ty, addr, ctx)
+    }
+
+    public(package) fun deny_list_v2_contains_next_epoch<T>(
```

Comment:
the lining up of this function's name + the `deny_list` name + receiver syntax + the other method name is just perfection.